static/test/javascripts/spec/common/commercial/adblock-messages.spec.js

```diff
@@ -0,0 +1,101 @@
+define([
+    'helpers/injector',
+    'common/utils/storage'
+], function (
+    Injector,
+    storage
+) {
+    var injector = new Injector();
+
+    describe('Adblock messages', function () {
```

Comment:
This looks like it's describing the messages themselves, as opposed to the service